### PR TITLE
[MacOS] Bleed in Background Image Fix and Image Size Fix

### PR DESCRIPTION
**Pull request by rohshar6**
## Description
1. Small Image Size Fix
2. Bleed Fix in Background Image

## Sample Card
Screenshot 2021-04-07 at 1 04 01 PM
Screenshot 2021-04-07 at 1 04 16 PM
Screenshot 2021-04-07 at 1 42 07 PM

## How Verified
How you verified the fix, including one or all of the following: 
1. New unit tests that were added if any. If none were added please add a quick line explaining why not.
2. Existing relevant unit/regression tests that you ran
3. Manual scenario verification if any; ***Do include .gif's or screenshots of the testing you performed here if you think that it 
will aid in code reviews or corresponding fixes on other platforms for eg.***

### CODE REVIEW

**Comment by mukuagar** [comment]
Does this fix not help in the horizontal line in top right of FlightDetails card?

**Comment by rohshar6** [comment]
> Does this fix not help in the horizontal line in top right of FlightDetails card?

No, Actually for that Manu is working on it

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift [hunk 1]
```diff
@@ -144,4 +147,19 @@ class ACRColumnView: ACRContentStackView {
             widthConstraint.isActive = false
         }
     }
+    
+    func bleedBackgroundImage(padding: CGFloat, top: Bool, bottom: Bool, leading: Bool, trailing: Bool, paddingBottom: CGFloat, with bottomAnchor: NSLayoutAnchor<NSLayoutYAxisAnchor>) {
+        backgroundImageTopConstraint.isActive = false
```
Comment (manujai) [inline source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift:152]: we can activate these new constraints without turning them false, right? 
Or do they have to be turned off and then activated again?

Comment (rohshar6) [reply source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift:152]: We have turn them off, otherwise getting conflicts of constraints error

Comment (manujai) [reply source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift:152]: Okay got it

**Comment by manujai** [comment]
> Does this fix not help in the horizontal line in top right of FlightDetails card?

This is fixed. I will add it in my next PR.

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift [hunk 2]
```diff
@@ -144,4 +147,19 @@ class ACRColumnView: ACRContentStackView {
...
+        backgroundImageViewLeadingConstraint.isActive = false
+        backgroundImageViewBottomConstraint.isActive = false
+        backgroundImageTopConstraint = backgroundImageView.topAnchor.constraint(equalTo: topAnchor, constant: top ? -padding : 0)
+        backgroundImageViewLeadingConstraint = backgroundImageView.leadingAnchor.constraint(equalTo: leadingAnchor, constant: leading ? -padding : 0)
+        backgroundImageViewTrailingConstraint = backgroundImageView.trailingAnchor.constraint(equalTo: trailingAnchor, constant: trailing ?  padding : 0)
+        backgroundImageViewBottomConstraint = backgroundImageView.bottomAnchor.constraint(equalTo: bottomAnchor, constant: bottom ?  padding : 0)
+        backgroundImageTopConstraint.isActive = true
+        backgroundImageViewTrailingConstraint.isActive = true
+        backgroundImageViewLeadingConstraint.isActive = true
+        backgroundImageViewBottomConstraint.isActive = true
```
Comment (blue-stone99) [inline source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift:163]:
```suggestion
    func bleedBackgroundImage(padding: CGFloat, top: Bool, bottom: Bool, leading: Bool, trailing: Bool, paddingBottom: CGFloat, with bottomAnchor: NSLayoutAnchor<NSLayoutYAxisAnchor>) {
          backgroundImageTopConstraint.constant = top ? -padding : 0
          backgroundImageViewLeadingConstraint.constant = leading ? -padding : 0
          backgroundImageViewTrailingConstraint.constant = trailing ?  padding : 0
          backgroundImageViewBottomConstraint.constant = bottom ? padding : 0
```

File: source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift [hunk 3]
```diff
@@ -49,6 +49,9 @@ class ACRColumnView: ACRContentStackView {
     
     private lazy var widthConstraint = widthAnchor.constraint(equalToConstant: Constants.minWidth)
     private lazy var backgroundImageViewBottomConstraint = backgroundImageView.bottomAnchor.constraint(equalTo: bottomAnchor)
+    private lazy var backgroundImageTopConstraint = backgroundImageView.topAnchor.constraint(equalTo: topAnchor)
```
Comment (blue-stone99) [inline source/macos/AdaptiveCards/AdaptiveCards/Views/ACRColumnView.swift:52]:
```suggestion
    private lazy var backgroundImageViewTopConstraint = backgroundImageView.topAnchor.constraint(equalTo: topAnchor)
```

**Pull request merged**